sklearn/linear_model/perceptron.py

```diff
@@ -123,6 +123,23 @@ class Perceptron(BaseSGDClassifier):
     ``Perceptron()`` is equivalent to `SGDClassifier(loss="perceptron",
     eta0=1, learning_rate="constant", penalty=None)`.
 
+    Examples
+    --------
+    >>> from sklearn.datasets import load_digits
+    >>> from sklearn.linear_model import Perceptron
+    >>> digits = load_digits()
+    >>> X = digits.data
+    >>> y = digits.target
+    >>> clf = Perceptron(alpha=0.0001, class_weight=None, fit_intercept=True,
```

Comment:
``clf = Perceptron(random_state=0)`` ?
You don't need to assign default value to parameters.